lib/Pandoc.pm

```diff
@@ -633,6 +633,13 @@ Benct Philip Jonsson
 
 =head1 LICENSE
 
-GNU General Public License, Version 2
+This software distribution is subject to the EUPL, Version 1.2 or subsequent
```

Comment:
Dist-zilla will probably add the license text on build, so one could suffice with just a comment in the source file: `# Covered by EUPL 1.2 or later. Please read the LICENSE file for more information.`